### DISCUSSION TITLE
How can I view delayed messages in the queue for arrival in the delay switch? thank you!

### DISCUSSION THREAD

**Discussion by 673161863-beinuan** [rabbitmq/rabbitmq-delayed-message-exchange#283] [General]
### Is your feature request related to a problem? Please describe.

How can I view delayed messages in the queue for arrival in the delay switch? thank you!

### Describe the solution you'd like

How can I view delayed messages in the queue for arrival in the delay switch? thank you!

**Comment by michaelklishin** [comment]
You cannot view messages but you can view a total number of them, it's an exchange metric exposed to the management UI.
